src/PerfView/PerfViewData.cs

```diff
@@ -4122,36 +4123,54 @@ protected internal override StackSource OpenStackSourceImpl(string streamName, T
             }
             else if (streamName == "CCW Ref Count")
             {
-                // TODO use the callback model.  We seem to have an issue getting the names however. 
-                foreach (var data in events.ByEventType<CCWRefCountChangeTraceData>())
+                var interopTraceEventParser = new InteropTraceEventParser(eventSource);
```

Comment:
The old code is still useful.     Rather than try to make a view that is 'both' we should instead simply make a new view called '.NET Native CCW Ref Count'.

Reply:
done